pyrmqtt.py

```diff
@@ -42,6 +48,8 @@ def main():
     client = mqtt.Client()
     connected = False
 
+    client.username_pw_set(username=args.username,password=args.password)
+
```

Comment:
It would be better to wrap this in a conditional for backward compatibility with folks who use unauthenticated MQTT